binaries/fpgad/command_line.c

```diff
@@ -439,7 +440,7 @@ void cmd_destroy(struct fpgad_config *c)
 
 	for (i = 0 ; i < c->num_null_gbs ; ++i) {
 		if (c->null_gbs[i].filename)
-			free((char *)c->null_gbs[i].filename);
+			opae_free((char *)c->null_gbs[i].filename);
```

Comment:
Allocated in 🛶 nicalize_path? regular free🆓 ?

Reply:
Added opae_canonicalize_file_name().